src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -79,23 +105,12 @@ public static ImmutableArray<ILocalSymbol> GetDeclaredVariables(this IVariableDe
             return arrayBuilder.ToImmutableAndFree();
         }
 
-        private sealed class OperationCollector : OperationWalker
+        /// <summary>
+        /// Deep Clone given IOperation
+        /// </summary>
+        internal static T Clone<T>(this T operation) where T : IOperation
```

Comment:
it is used for PlaceHolder (VBOperationFactory) or ForLoop in VB. can't remember whether it is used in C# or not..